htdocs/product/stock/card.php

```diff
@@ -608,8 +609,10 @@
 
 			// TODO Create $arrayfields with all fields to show
 
-			print '<table class="noborder centpercent">';
-			print "<tr class=\"liste_titre\">";
+			print '<div class="div-table-responsive">';
```

Comment:
Can you move this div just under the print of title (1 line toward bottom) ?

Reply:
done.  and force push
I hope I understood you right.